src/lib.rs

```diff
@@ -551,6 +551,11 @@ where
         Ok(id)
     }
 
+    pub fn get_native_texture(&self, id: ImageId) -> Option<T::NativeTexture> {
```

Comment:
I think it would be nice to have some documentation. (I know get_image doesn’t, but trying to start a counter-trend ;)

As a caller I’d like to know what it means when this fails or returns None.

Perhaps this function should return a Result instead of Option?

Reply:
No problem, I'll add some docs. And good idea on making it return a result.